src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,601 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Transforms.TensorFlow;
+
+[assembly: LoadableClass(TensorflowTransform.Summary, typeof(TensorflowTransform), typeof(TensorflowTransform.Arguments), typeof(SignatureDataTransform),
+    TensorflowTransform.UserName, TensorflowTransform.ShortName)]
+
+[assembly: LoadableClass(TensorflowTransform.Summary, typeof(TensorflowTransform), null, typeof(SignatureLoadDataTransform),
+    TensorflowTransform.UserName, TensorflowTransform.LoaderSignature)]
+
+namespace Microsoft.ML.Transforms
+{
+    public sealed class TensorflowTransform : RowToRowMapperTransformBase
+    {
+        public sealed class Column : ManyToOneColumn
```

Comment:
>ManyToOneColumn [](start = 37, length = 15)

Consider not using this base class (or the ManyToOneColumnBindingsBase class). These are intended for applying the transform simultaneously to multiple sets of input columns. It is advantageous in case the transform requires training (such as ngram transform) because defining multiple transforms would require multiple passes over the data.